src/admin/index.js

```diff
@@ -270,8 +270,25 @@ module.exports = ({
 
         await cluster.refreshMetadata()
       } catch (e) {
-        if (['NOT_CONTROLLER', 'UNKNOWN_TOPIC_OR_PARTITION'].includes(e.type)) {
-          logger.warn('Could not delete topics', { error: e.message, retryCount, retryTime })
+        if (
+          e.name === 'KafkaJSAggregateError' &&
+          e.errors.some(error => error.type.includes('UNKNOWN_TOPIC_OR_PARTITION'))
```

Comment:
This should be `error => error.type === 'UNKNOWN_TOPIC_OR_PARTITION'`.  The use of includes in the original code was on an Array. But here you're using it on a string, which works but I think only accidentally, and I'm pretty sure is not what you intend.

Reply:
This one is also changed.

Thank you.